docs/index.md

```diff
@@ -255,6 +255,40 @@ Please follow these simple steps:
 </html>
 ```
 
+#### Viewing distant objects in location-based mode 
+
+If your location-based AR content is distant from the user (around 1km or
+more), it is recommended to use the new `arjs-webcam-texture` component (introduced in AR.js 3.2.0), which uses a THREE.js texture to stream the camera feed and allows distant content to be viewed. This component is automatically injected if the `videoTexture` parameter of the `arjs` system is set to `true` *and* the `sourceType` is `webcam`. For example (code snippet only):
+
+```html
+    <a-scene
+      vr-mode-ui="enabled: false"
+      embedded
+      arjs="sourceType: webcam; videoTexture: true; debugUIEnabled: false;"
+    >
+```
+
+#### Reducing shaking effects in location-based mode
+
+In location-based mode, 'shaking' effects can occur due to frequent small
+changes in the device's orientation, due to the high sensitivity of the device 
+sensors such as the accelerometer. If using AR.js 3.3.1 or greater this can optinally be reduced using an exponential smoothing technique. *Note that there are currently some occasional display artefacts with this if moving the device quickly or suddenly so please test before you enable it in a finished application; work to resolve these is on-going.*
+
+This is enabled by adding the `arjs-look-controls` component to your `a-camera` with a `smoothingFactor` property. You must also disable A-Frame's default `look-controls`, as `arjs-look-controls` will replace it. For example:
+
+```html
+<a-camera id='camera1' look-controls-enabled='false' near="0.1" far="50000" fov="80" arjs-look-controls='smoothingFactor: 0.1' gps-projected-camera='gpsMinDistance: 5' rotation-reader vertical-controls wasd-controls="acceleration: 1300"> </a-camera>
```

Comment:
I would specify only the needed parameters, so:

```
<a-camera look-controls-enabled='false' arjs-look-controls='smoothingFactor: 0.1' gps-camera='gpsMinDistance: 5;' rotation-reader> </a-camera>
```

Reply:
I would add also this part:

```
Also, you can reduce shaking  when near a place - a bad-looking effect due to GPS sensors low precision.
To do so you can use the `gpsMinDistance` and `gpsTimeInterval ` properties.
Look to the <a href="./location-based.md"> Location Based specific doc </a> to learn how to use them.
```

Reply:
Good points.